docs/rbac-permissions-operators.yaml

```
@@ -12,7 +12,7 @@ rules:
   resources: ["podsecuritypolicies", "roles", "rolebindings"]
   verbs: ["create", "delete"]
 - apiGroups: [""]
-  resources: ["configmaps"]
+  resources: ["configmaps", "nodes"]
```

```suggestion
  resources: ["configmaps"]
```
Since we are not accessing the nodes anymore, we can remove this from the RBAC manifest

---

I don't have an opinion here on this. What's this about?

---

I think this change was needed earlier in this PR to allow the charms to query Node resources and get their k8s version. I agree with Stone's suggestion, now that the charms determine PSP availability via the k8s API, they're no longer querying nodes, so this line can be reverted.